jdi-light-parallel-examples/src/main/java/io/github/epam/EpamGithubSite.java

```diff
@@ -0,0 +1,28 @@
+package io.github.epam;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import io.github.epam.pages.HomePage;
+import io.github.epam.pages.PerformancePage;
+import io.github.epam.sections.Header;
+
+@JSite("https://jdi-testing.github.io/jdi-light/")
+public class EpamGithubSite {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'EpamGithubSite' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)